app/src/main/java/de/westnordost/streetcomplete/quests/AbstractOsmQuestForm.kt

```diff
@@ -241,7 +241,12 @@ abstract class AbstractOsmQuestForm<T> : AbstractQuestForm(), IsShowingQuestDeta
 
     protected fun composeNote() {
         val questTitle = englishResources.getQuestTitle(osmElementQuestType, element.tags)
-        val leaveNoteContext = "Unable to answer \"$questTitle\""
+        val hintLabel = getNameAndLocationLabel(element, englishResources, featureDictionary)
+        val leaveNoteContext = if (hintLabel.isNullOrBlank()) {
+            "Unable to answer \"$questTitle\""
+        } else {
+            "Unable to answer \"$questTitle – $hintLabel\""
```

Comment:
Wrong placement of `"`?

Should it really be

> Unable to answer "Are these opening hours still correct? - on level 2: Mobile Service (Mobile Phone Shop)"

and not rather

> Unable to answer "Are these opening hours still correct?" - on level 2: Mobile Service (Mobile Phone Shop)

?

Reply:
I don't really have a preference here, if you like like the second variant more I'll change it.

Reply:
Yeah, I like the second option more, or actually, the full string would be 

> Unable to answer "Are these opening hours still correct?" - on level 2: Mobile Service (Mobile Phone Shop) for https://osm.org/node/....

wouldn't it? Actually, my suggestion was to not use "for" at all, but use "-", i.e.

> Unable to answer "Are these opening hours still correct?" - on level 2: Mobile Service (Mobile Phone Shop) - https://osm.org/node/....